charts/intel-gpu-resource-driver/templates/resource-driver-namespace.yaml

```diff
@@ -1,4 +1,6 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: intel-gpu-resource-driver
+  name: {{ include "intel-gpu-resource-driver.namespace" . }}
+  labels:
+    pod-security.kubernetes.io/enforce: privileged
```

Comment:
Privileged namespace security is required to mount host path files, so if the Chart managed the namespace it should by default set the required permissions. Let me know if this should be templated instead, I personally don't think so since it's a requirement for the pod to start.

Reply:
Thanks, that's good to know!

The annotation seems to be documented e.g. here: https://kubernetes.io/docs/tasks/configure-pod-container/enforce-standards-namespace-labels/